src/Mod/PartDesign/Gui/TaskExtrudeParameters.cpp

```diff
@@ -108,6 +108,8 @@ void TaskExtrudeParameters::setupDialog()
 
     // Fill data into dialog elements
     // the direction combobox is later filled in updateUI()
+    ui->lengthEdit->setMinimum(-DBL_MAX);
+    ui->lengthEdit2->setMinimum(-DBL_MAX);

```

Comment:
All the changes to TaskExtrudeParameters.cpp are unecessary.
You need only change the TaskPadPocketParameters.ui where Length has a minimum defined : 



```
     <item row="1" column="1">
      <widget class="Gui::PrefQuantitySpinBox" name="lengthEdit" native="true">
       <property name="keyboardTracking" stdset="0">
        <bool>false</bool>
       </property>
       <property name="unit" stdset="0">
        <string notr="true">mm</string>
       </property>
       <property name="minimum" stdset="0">
        <double>0.000000000000000</double>
       </property>
      </widget>
     </item>
```

You just need to remove 
```
       <property name="minimum" stdset="0">
        <double>0.000000000000000</double>
       </property>
```